.github/ISSUE_TEMPLATE.md

```diff
@@ -4,6 +4,8 @@ This issue tracker is only for technical issues related to bitcoin-core.
 
 General bitcoin questions and/or support requests and are best directed to the [Bitcoin StackExchange](https://bitcoin.stackexchange.com).
 
+For reporting security issues, please use [security@bitcoincore.org](mailto:security@bitcoincore.org).
```

Comment:
At wanted to do so first, but the URLs are fragile and can change. And when the separate "Report security issues" page happen at bitcoincore.org, we will have to change URL here. Mail will probably be the same.